### ISSUE TITLE
create-falcon-app, does not support filepaths with spaces

### ISSUE THREAD

**Issue by TarVK** [deity-io/falcon#338]
**Describe the bug**
When the filepath in which the app creation is run contains a space, the setup errors.

**To Reproduce**
Steps to reproduce the behavior (option a):
1. Create a directory that contains a space in the name
2. Run `npx create-falcon-app my-app` in this directory
3. Observe the setup failing, by means of running an unknown command

Steps to reproduce the behavior (option b):
1. Create a directory that contains a space in the name
2. Run `yarn create falcon-app my-app` in this directory
3. Observe the setup failing, by means of running an unknown command

**Expected behavior**
The setup should finish successfully by creating a client and server directory.

**Screenshots**
(different directory of path contains spaces)
![image](https://user-images.githubusercontent.com/26687938/54156679-05ab7400-4447-11e9-9f3c-5efc6394e8bf.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - NodeJS version: 10.15.3
 - NPM version: 6.4.1

**Additional context**
I was able to 'get around' the issue by first calling the yarn setup;

`yarn create falcon-app my-app`

followed by the npx setup:

`npx create-falcon-app my-app`

but neither seem to work individually.
Yarn seems to be able to finish most of the setup process, after which the npx call seems to be able to skip the step on which it errored without causing problems. This makes me think that the problem might be within the dependencies rather than the tool itself. But that's just a hunch.

**Comment by TarVK** [comment]
I performed some tests with the react create app system for comparison. 
This seems to suffer from mostly the same problems. 
Both their npx and yarn setup don't work straight away, but npx works after having called yarn.

Here are pictures of commands I performed to create a react app, in the correct sequence:
![image](https://user-images.githubusercontent.com/26687938/54561668-731a5000-49c5-11e9-92a3-b0ad609e96f5.png)
![image](https://user-images.githubusercontent.com/26687938/54561683-7b728b00-49c5-11e9-9872-c61231a5c1a4.png)
![image](https://user-images.githubusercontent.com/26687938/54561701-85948980-49c5-11e9-90a9-8973552157eb.png)
